systemtest/src/test/java/io/strimzi/systemtest/metrics/MetricsST.java

```diff
@@ -228,33 +236,30 @@ void testClusterOperatorMetrics() {
             assertCoMetricNotNull("strimzi_reconciliations_failed_total", resource, clusterOperatorMetricsData);
         }
```

Comment:
should we add the new one `strimzi_resource_state` to the check in this list as well?

Reply:
Ye I can add it

Reply:
I checked the code and it kinda doesn't make sense to add it into this loop. Basically if the metric is set properly is tested by new asserts which I added after this loop.